bigtable/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 [1]: https://pypi.org/project/google-cloud-bigtable/#history
 
+## 0.31.0
+
+### Implementation Changes
+- Improve admin operation timeouts. ([#6010](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6010))
+
+### New Features
+
+### Dependencies
+
+### Documentation
+- Cloud Bigable readiness upgraded from alpha to beta.
+
+### Internal / Testing Changes
+- Preparing Cloud Bigtable for beta ([#6129](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6129))
+- Harden instance teardown against '429 Too Many Requests'. ([#6102](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6102))
+- Bigtable: prep docs for repo split. ([#6014](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6014))
+- Bigtable: Add '{RowSet,RowRange}.{__eq__,.__ne__}' ([#6025](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6025))
+- Re-generate library using bigtable/synth.py ([#6036](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/6036))
+
```

Comment:
We need an entry for #6137, since it has been merged.

Reply:
Thanks for adding it!